build/Palaso.proj

```diff
@@ -86,14 +87,24 @@
 			Condition="'$(teamcity_version)' != ''"/>
 		<Copy SourceFiles="@(NUnitAddinFiles)" DestinationFolder="$(NUnitRunnerPackage)/tools/addins"
 			Condition="'$(teamcity_version)' != ''"/>
-		<NUnit Assemblies="@(TestAssemblies)"
-			ToolPath="$(NUnitRunnerPackage)/tools"
-			TestInNewThread="false"
+		<NUnit3 Condition="'$(teamcity_version)' == ''"
+			Assemblies="@(TestAssemblies)"
+			ToolPath="$(RootDir)/packages/NUnit.ConsoleRunner.3.10.0/tools"
 			ExcludeCategory="$(ExtraExcludeCategories)$(excludedCategories)"
 			WorkingDirectory="$(RootDir)/output/$(Configuration)"
 			Force32Bit="$(useNUnit-x86)"
 			Verbose="true"
-			OutputXmlFile="$(RootDir)/output/$(Configuration)/TestResults.xml"/>
+			OutputXmlFile="$(RootDir)/output/$(Configuration)/TestResults.xml"
+      UseNUnit3Xml = "true"
+			TeamCity="false"/>
+		<NUnit3 Condition="'$(teamcity_version)' != ''"
```

Comment:
It seems that it should be possible to combine the two TC and non-TC calls since they are basically identical.

Reply:
Maybe so. I copied this pretty much verbatim from HearThis. I'll have a look.

Reply:
I removed one of the three differences ("SkipOnTeamCity" was already in ExtraExcludeCategories, when run on TC). Concerning the other two differences: The TeamCity true/false should be straightforward, but I wasn't sure how to leave OutputXmlFile undefined for the TC build.

Reply:
Does it matter if we generate a TestResults.xml file on TC? We don't need it there, but it probably doesn't matter if it exists.